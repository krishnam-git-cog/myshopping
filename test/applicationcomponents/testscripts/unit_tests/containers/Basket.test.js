import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import Basket from '../../../../../src/containers/basket/Basket';

describe('Basket test', () => {
    it("Container - Basket should match snapshot", () => {
        const productCodes = '["foo"]';
        localStorage.setItem('items', JSON.stringify(productCodes));
        let shallow = createShallow();
        const component = shallow(<Basket />);
        expect(component).toMatchSnapshot();
    });
});