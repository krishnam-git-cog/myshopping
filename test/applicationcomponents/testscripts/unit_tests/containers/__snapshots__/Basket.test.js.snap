// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basket test Container - Basket should match snapshot 1`] = `ShallowWrapper {}`;
